.github/workflows/drc_sc7_test.yml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 GlobalFoundries PDK Authors
```

Comment:
`sc7` should be the proper name for the library.

Also, it seems to me that the DRC checking for the standard cell library should happen in the standard cell library repo @ https://github.com/google/globalfoundries-pdk-libs-gf180mcu_fd_sc_mcu7t5v0 ?

Also comments from other workflows.

Reply:
For the naming, Okay it's my fault.

For DRC testing on standard cell library, we will have to copy the rule deck and scripts that run DRC on them and edit the testing script because all standard cells here are in one GDS file.

Reply:
I think we need to discuss this part a bit more.